webview/cocoa.py

```diff
@@ -348,6 +348,21 @@ def toggle():
         AppHelper.callAfter(toggle)
         self.is_fullscreen = not self.is_fullscreen
 
+    def set_window_size(self, width, height):
+        def _set_window_size():
+            frame = self.window.frame()
+
+            # Keep the top left of the window in the same place
+            frame.origin.y += frame.size.height;
+            frame.origin.y -= height;
```

Comment:
semicolons

Reply:
Hah- I guess I just copy pasted from Obj-C code XD

Reply:
Fixed it though